README.md

```diff
@@ -81,10 +81,44 @@ Finally, we look at the procedural steps adopted by the framework:
 
 1. The JSON configuration file is read and parsed, and any default field values are assigned. If the configuration file is missing, the program throws a fatal error.
 2. Experiment service times (e.g., 10 seconds) are translated on the client machine into numbers representing busy-spin increment limits (e.g., 10,000,000). In turn, those are used by the measurement function on the server machine to keep the processor busy-spinning.
-3. A connection with the serverless vendor is established.  This is abstracted away behind a common interface having only four functions: ListAPIs, DeployFunction, RemoveFunction, and UpdateFunction. Used exclusively throughout the codebase, this interface offers seamless integration functionality with any provider.
-4. In the provisioning phase, existing endpoints are first queried either using official provider APIs or from a local file.  The corresponding serverless functions are then updated, deployed, or removed to match the specified configuration file.
+3. A connection with the serverless vendor is established. This is abstracted away behind a common interface having only four functions: ListAPIs, DeployFunction, RemoveFunction, and UpdateFunction. Used exclusively throughout the codebase, this interface offers seamless integration functionality with any provider.
+4. In the provisioning phase, serverless.com framework is used to deploy the functions to the cloud and to establish HTTP endpoints. The functions are configured based on the experiment JSON file. 
 5. The last step runs all the experiments either sequentially or in parallel: bursts are successively sent to each available endpoint, followed by a sleep duration specified by the IAT. The process is repeated until all responses have been recorded to disk. Finally, statistics and visualizations are generated.
 
+## Serverless.com Framework Deployment
+
+Stellar uses serverless.com framework to deploy serverless functions to cloud.
+
+![deployment](design/deployment.png)
+
+1. The JSON experiment configuration file is parsed and serverless.yml service configuration file is written. 
+2. The function source code is compiled if needed. (e.g. Java and Go functions)
+3. The filler file is created to increase the function deployment size.
```

Comment:
Shall we jot down a few words on what a filler file is & why we use it. 
Since we are mentioning the term "_filler file_" for the first time, a user might get confused imo.

Reply:
I added the explenation.